### DISCUSSION TITLE
SMARTer Mouse Bcell repertoire analysis

### DISCUSSION THREAD

**Discussion by PaulDanPhillips** [milaboratory/mixcr#1327] [Q&A] [answered]
Hello,
I'm curious as to why specialized kit commands (SMARTer Human BCR, SMARter Mouse TCR and even the in-house Milabs kits) are incomplete. Why not have the analyze takara-mouse-rna-bcr command to complete the SMARTer Mouse data analysis?

While this does not exist and may not be necessary, I was hoping someone could help me define the --tag-pattern option for a SMARTer mouse BCR kit (linked below and does not contain umi's) using the mixcr analyze generic-amplicon command.

(https://www.takarabio.com/documents/User%20Manual/SMARTer%20Mouse%20BCR%20IgG%20H/SMARTer%20Mouse%20BCR%20IgG%20H-K-L%20Profiling%20Kit%20User%20Manual_121217.pdf) 

Thanks!

**Comment by mizraelson** [comment, marked as answer]
Hi, what do you mean by 'incomplete'? The pattern is built into every specialized command.

For the SMARTer mouse BCR kit, you can use the command below. No need to specify anything else.

```
mixcr analyze takara-mouse-rna-bcr-smarter \
      input_R1.fastq.gz \
      input_R2.fastq.gz \
      result 
```

Note that this kit does not include UMIs, so the pattern only trims the TSO sequence.

**Reply by PaulDanPhillips** [reply]
Thanks so much. I was going off of the guides section in the documentation, but I appreciate that it would be unnecessary to provide a guide for every single kit.
